app/assets/javascripts/3dbio_viewer/src/webapp/components/network/NetworkForm.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useState, useRef } from "react";
+import i18n from "../../utils/i18n";
+import { Dropzone, DropzoneRef } from "../dropzone/Dropzone";
+import IncludeNeighborsCheckbox from "./IncludeNeighborsCheckbox";
+import Label from "./Label";
+import NetworkExample from "./NetworkExample";
+import UniProtAccessionTextArea from "./UniProtAccessionTextArea";
+import SpeciesSelect from "./SpeciesSelect";
+import "./Network.css";
+
+interface NetworkForm {
+    species: string;
+    uniProtAccession: string;
+    includeNeighboursWithStructuralData: boolean;
+}
+
+function getInitialNetworkForm(): NetworkForm {
+    return {
+        species: "homoSapiens",
+        uniProtAccession: "",
+        includeNeighboursWithStructuralData: false,
+    };
+}
+
+const NetworkForm = React.memo(() => {
+    const annotationFileRef = useRef<DropzoneRef>(null);
+    const [error, setError] = useState<string>();
+    const [networkForm, setNetworkForm] = useState<NetworkForm>(() => getInitialNetworkForm());
```

Comment:
`() => fn()` -> `fn` (Caveat: unless `fn` is in fact a class-method that carries a bound context for `this`!)